build.sbt

```diff
@@ -7,8 +7,10 @@ val sttpVersion = "1.6.3"
 val Specs2Version = "4.2.0"
 val artifactory = "https://cognite.jfrog.io/cognite/"
 val cogniteSdkVersion = "1.2.1"
+val prometheusVersion = "0.8.1"
+val log4sVersion = "1.8.2"
 
-resolvers += "libs-release" at artifactory + "libs-release/"
+resolvers += "libs-release".at(artifactory + "libs-release/")
```

Comment:
Not sure if it's your IntelliJ or if we need to change some config file to exclude build.sbt, but for better or worse this just isn't the formatting used for SBT files, and it's inconsistent with most of the rest of the file. The same applies for several other changes in this diff.

Reply:
Yes saw it. I thought maybe this was something you wanted since we don't ignore sbt files from formatting. We should exclude it from scalafmt.

Reply:
Okey, it was just me using a new intellij version, and had to set scalafmt as formatter again.